src/main/java/net/spy/memcached/ConnectionFactoryBuilder.java

```diff
@@ -357,6 +357,15 @@ public NodeLocator createLocator(List<MemcachedNode> nodes) {
           return new ArrayModNodeLocator(nodes, getHashAlg());
         case CONSISTENT:
           return new KetamaNodeLocator(nodes, getHashAlg());
+        case ROUND_ROBIN:
+          if (getFailureMode() != FailureMode.Cancel
```

Comment:
@amcrn what is the reason for excluding Redistribute FailureMode ? RoundRobin seems like a good fit for Redistribute as well.